makeit/mconfig

```diff
@@ -18,6 +20,9 @@ tgtcxx=
 tgtcxx_opts=$hstcxx_opts
 hststatic=0
 tgtstatic=0
+package_version=`git describe --match 'v[0-9]*' --dirty --always 2>/dev/null || echo "" | sed -e "s/^v//;s/-/_/g;s/_/-/;s/_/./g"`
+# export so this can be used in mlocal/checks
```

Comment:
All vars above including package_version don't need to be exported with how the build system embeds other units. We can safely remove lines 24 and 25. Cheers.

Reply:
Ok I removed the export.